### DISCUSSION TITLE
Bug report

### DISCUSSION THREAD

**Discussion by OkuruchiDan** [bibixx/react-adobe-animate#70] [Q&A / Help] [answered]
Hello there! I had found a bug 
I'm trying to animate local js file with this library using `NEXT.JS`. So you can try to reproduce this bug on your own project! 

### Step by step reproducing on stage
1. Go to [Staging landing page](https://staging.perfsol.tech/ ).
2. Go to any other page, for example [Blog page](https://staging.perfsol.tech/blog) or one of [Case-studies](https://staging.perfsol.tech/case-studies/3-twinbody).
3. Refresh the page(using refresh button or hot keys, no matter).
4. Click on back button in your browser(should redirect you to '/' landing page). 
5. Tada!!! you have an **_Error_** :)
6. Open console to see the logs of your library.

Any Ideas how to fix it? 
I had created also an Issue about that 
Here is link -> https://github.com/bibixx/react-adobe-animate/issues/69

**Comment by OkuruchiDan** [comment]
@bibixx Hello, any ideas about this behaviour?

**Comment by bibixx** [comment]
@OkuruchiDan Hi, sorry for the late response. You know how it is before Christmas 😉
I'll _try_ to look into it tomorrow/monday, but cannot promise anything as this package has always had problems with Next.js projects and I think it may need a bit of rewrite (which I think should happen either way)

**Comment by bibixx** [comment, marked as answer]
Further discussion should be done in #69

**Reply by OkuruchiDan** [reply]
okay, thanks!)
